### PR TITLE
Added cloudPcFrontlineInsufficientLicensesScenario in alertRuleTemplate values

### PR DESCRIPTION
**Pull request by zansy**
Added cloudPcFrontlineInsufficientLicensesScenario in alertRuleTemplate values
the CSDL and api.md files for the PR: 
- [Add cloudPcFrontlineInsufficientLicensesScenario scenario to alertRuleTemplate](https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/13980)
- [Changed details type in alertRecordTemplate](https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/14267)

### CODE REVIEW

File: api-reference/beta/resources/devicemanagement-alertimpact.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Represents the impact of an alert event.
 
 |Property|Type|Description|
 |:---|:---|:---|
+|alertImpactDetails|[keyValuePair](../resources/keyvaluepair.md) collection|The detail information of the impact. For example, when trigger alert `Frontline Cloud PCs near concurrency limit`, the details would contain impacted Frontline license SKU name like `Windows 365 Frontline 2 vCPU/8GB/128GB` and the corresponding impacted value. |
```
Comment (JarbasHorst) [inline api-reference/beta/resources/devicemanagement-alertimpact.md:24]: For clarity, I recommend the following revision for this description:
- For example, if the \_Frontline Cloud PCs near concurrency limit\_ alert is triggered, the details contain the impacted frontline license SKU name, such as \`Windows 365 Frontline 2 vCPU/8GB/128GB\` and the corresponding impacted value.

**Comment by JarbasHorst** [comment]
Hello @zansy, I reviewed your PR and left one comment. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [22133](https://github.com/microsoftgraph/microsoft-graph-docs/pull/22133) into your branch to include my fixes and before addressing comment.

File: api-reference/beta/resources/devicemanagement-alertimpact.md (hunk 1 above)
Comment (zansy) [reply api-reference/beta/resources/devicemanagement-alertimpact.md]: Resolved, modified as suggested. Thank you!

**Comment by JarbasHorst** [comment]
Hi @Lauragra this PR is ready for final review. Please, merge PR [21642](https://github.com/microsoftgraph/microsoft-graph-docs/pull/22214) into this branch to include my final edits.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**